packages/okta-react-native/index.js

```diff
@@ -0,0 +1,94 @@
+import { NativeModules, Platform } from 'react-native';
+import {
+	assertIssuer,
+  assertClientId,
+	assertRedirectUri
+} from '@okta/configuration-validation';
+import jwt from 'jwt-lite';
+
+export const createConfig = async (
+  clientId,
+  redirectUri, 
+  endSessionRedirectUri, 
+  discoveryUri,
+  scopes) => {
+
+  assertIssuer(discoveryUri);
+  assertClientId(clientId);
+  assertRedirectUri(redirectUri);
+  assertRedirectUri(endSessionRedirectUri);
+
+  if (Platform.OS === 'ios') {
+		scopes = scopes.join();
+		scopes = scopes.replace(/,/g, ' ');
+  }
+    
+  return NativeModules.OktaSdkBridge.createConfig(
+    clientId,
+    redirectUri,
+    endSessionRedirectUri,
+    discoveryUri,
+    scopes
+  );
+} 
+
+export const signIn = async () => {
+  return NativeModules.OktaSdkBridge.signIn();
+}
+
+export const signOut = async () => {
+  return NativeModules.OktaSdkBridge.signOut();
+}
+
+export const getAccessToken = async () => {
+  return NativeModules.OktaSdkBridge.getAccessToken();
+}
+
+export const getIdToken = async () => {
+  return NativeModules.OktaSdkBridge.getIdToken();
+}
+
+export const getUser = async () => {
+  return NativeModules.OktaSdkBridge.getUser();
+}
+
+export const getUserFromIdToken = async() => {
+  try {
+    let idTokenResponse = await getIdToken();
+    return jwt.decode(idTokenResponse.id_token).claimsSet;
+  } catch(error) {
+    throw error;
+  } 
```

Comment:
Why the try/catch if the behavior will be the same as without the try catch?

Reply:
I was thinking to catch the error from `let idTokenResponse = await getIdToken();`, would that be the same with removing try and catch?

Reply:
You do catch it, but then immediately throw it when caught.  So with or without the try/catch, the error will be thrown.